test/integration/ctl/ctl_status_certificate_test.go

```diff
@@ -114,18 +121,20 @@ No CertificateRequest found for this Certificate`,
 			certificate: gen.Certificate(crt2Name,
 				gen.SetCertificateNamespace(ns1),
 				gen.SetCertificateDNSNames("www.example.com"),
-				gen.SetCertificateIssuer(cmmeta.ObjectReference{Name: "letsencrypt-prod", Kind: "ClusterIssuer"}),
+				gen.SetCertificateIssuer(cmmeta.ObjectReference{Name: "letsencrypt-prod", Kind: "Issuer"}),
 				gen.SetCertificateSecretName("example-tls")),
 			certificateStatus: &cmapi.CertificateStatus{Conditions: []cmapi.CertificateCondition{crtReadyAndUpToDateCond, crtIssuingCond},
 				NotAfter: &metav1.Time{Time: certIsValidTime}, Revision: &revision1},
 			inputArgs:      []string{crt2Name},
 			inputNamespace: ns1,
 			req:            req1,
 			reqStatus:      &cmapi.CertificateRequestStatus{Conditions: []cmapi.CertificateRequestCondition{reqNotReadyCond}},
-			expErr:         false,
-			// Note: after `Event:` there is actually a tab, which sometimes shows as a single space, sometimes multiple
+			issuer: gen.Issuer("letsencrypt-prod",
+				gen.SetIssuerNamespace(ns1)),
+			expErr: false,
 			expOutput: `Name: testcrt-2
 Namespace: testns-1
+Created at: ([0-9]+)-(0[1-9]|1[012])-(0[1-9]|[12][0-9]|3[01])[Tt]([01][0-9]|2[0-3]):([0-5][0-9]):([0-5][0-9]|60)(\.[0-9]+)?(([Zz])|([\+|\-]([01][0-9]|2[0-3]):[0-5][0-9]))
```

Comment:
so euh... can I talk you into using a fake clock for these :sweat_smile:

Reply:
I tried to find something but all of the fake clocks I found are used by passing them around, which doesn't seem to be what I need.

Reply:
@meyskens agreed on the stand-up to leave it as is since the creationTimestamp seems to be set by the API server.